src/chain/Makefile

```diff
@@ -40,8 +40,7 @@ ifeq ($(CUDA), true)
   #For toolkit older than 6.5, add the compute capability 1.0
   CUDA_VER_GT_6_5 := $(shell [ $(CUDA_VERSION) -ge 65 ] && echo true)
   ifneq ($(CUDA_VER_GT_6_5), true)
-    CUDA_ARCH += -gencode arch=compute_13,code=sm_13 \
-                 -gencode arch=compute_10,code=sm_10
+    CUDA_ARCH += -gencode arch=compute_13,code=sm_13
```

Comment:
THe compXchg and compCAS are defined since compute 1.1 or 1.2 (not sure now), but definitely not 1.0